### DISCUSSION TITLE
Nous A1T: show voltage always

### DISCUSSION THREAD

**Discussion by GerdRuetten** [arendst/Tasmota#18620] [Q&A] [answered]
Dear all,

i flashed two A1T with Tasmota 12.5.0 DE

The voltage value only show, if the device is "on"

If it "off" it shows 0 voltage.

Is it possible to show the voltage always?

TIA

Best regards

Gerd

**Comment by barbudor** [comment, marked as answer]
https://tasmota.github.io/docs/Commands/#setoption21

**Reply by GerdRuetten** [reply]
Ok Note to myself: rtfm to last side_
